src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * Ui class to deal with user interaction
+ */
+public class Ui {
+    private static final Scanner sc = new Scanner(System.in);
+    private void showLine() {
+        System.out.println("   __________________________________________________________________");
+    }
+
+    /**
+     * Returns next line of user input
+     *
+     * @return Next line of user input
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints welcome message
+     */
+    public void showWelcome() {
+        showLine();
+        System.out.println("    Hello, I'm Duke");
+        System.out.println("    I can help you keep track of all your tasks! ☆*:.｡.o(≧▽≦)o.｡.:*☆");
+        showLine();
+        System.out.println("    How to add tasks to the list:");
+        System.out.println("    ToDo - type 'todo' followed by the description");
+        System.out.println("    Deadline - type 'deadline' followed by the description,");
+        System.out.println("    then '/by', then due date in yyyy-MM-dd format");
+        System.out.println("    Event - type 'event' followed by the description,");
+        System.out.println("    then '/at', then timing in yyyy-MM-dd format");
+        System.out.println("    Type 'done' followed by the task number " +
+                "and I'll mark it as done");
+        System.out.println("    Type 'list' to see the list");
+        System.out.println("    Type 'find' followed by keyword to search for tasks");
+        System.out.println("    Type 'bye' to exit");
+        showLine();
```

Comment:
May be more efficient speed wise to concat every line and make it into a constant variable.